debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresSchemaIT.java

```diff
@@ -370,6 +374,91 @@ public void shouldPopulateToastableColumnsCache() throws Exception {
         }
     }
 
+    @Test
+    public void shouldProperlyGetDefaultColumnValues() throws Exception {
+        String ddl = "DROP TABLE IF EXISTS default_column_test; CREATE TABLE default_column_test (" +
+                "pk SERIAL, " +
+                "bigint BIGINT default 9223372036854775807, " +
+                "bit_as_boolean BIT(1) default B'1', " +
+                "bit BIT(2) default B'11', " +
+                "varbit VARBIT(5) default B'110', " +
+                "boolean BOOLEAN not null default TRUE, " +
+                // box
+                // bytea
+                "char CHAR(10) default 'abcd', " +
+                "varchar VARCHAR(100) default 'abcde', " +
+                // cidr
+                // date
+                "double float8 default 123456789.1234567890123, " +
+                // inet
+                "integer INT default 2147483647, " +
+                // interval
+                "json JSON default '{}', " +
+                "jsonb JSONB default '{}', " +
+                // line
+                // lseg
+                // macaddr
+                // macaddr8
+                // money
+                "numeric NUMERIC(10, 5) default 12345.67891, " +
+                // path
+                // pg_lsn
+                // point
+                // polygon
+                "real FLOAT4 default 1234567890.5, " +
+                "smallint INT2 default 32767, " +
+                "text TEXT default 'asdf', " +
+                // time
```

Comment:
Most notably the date & time related types are omitted here.  At least in my case they're all expressions using either `now()` or `current_timestamp`, which I don't _think_ will translate well as default values.  If desired I can add code to at least attempt to parse the provided default in case it is a valid date and let it fail otherwise.

Reply:
Yes, if an actual value is provided, it'd be good to propagate this (I believe we do this for SQL Server, too). `now()` and related non-absolute values are problematic of course, I believe we export the equivalent to zero in this case for SQL Server. @jpechane, can you confirm?